src/uct/ib/ud/base/ud_iface.c

```diff
@@ -958,3 +958,9 @@ void uct_ud_iface_progress_enable(uct_iface_h tl_iface, unsigned flags)
 
     uct_base_iface_progress_enable(tl_iface, flags);
 }
+
+void uct_ud_iface_send_completion(uct_ud_iface_t *iface, uint16_t sn,
+                                  int is_async)
+{
+    /* Unimplemented for now */
```

Comment:
what is the plan?
just checking regarding the am pending ordering

Reply:
it does not affect pending queue logic. in addition to increasing send resources in mlx5/verbs, this function would also release some control skb's which could, in turn, call some zero-copy completion callback. all that happens after the message has already been sent.

Reply:
yes, but can zero-copy completion callback call some tx func?
in this case it would be handled before pending queue

Reply:
yes, good point